servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -64,6 +66,9 @@ def create_app(self):
             app = connexion.App(__name__, specification_dir='../swagger/')
             app.app.json_encoder = JSONEncoder
             app.add_api('swagger.yaml')
+            app.app.config[
+                'AGGREGATION_FILTER_LABEL'] = "aggregation-testing-unique:" + datetime.datetime.now(
+                ).strftime('%Y%m%d_%H%M')
```

Comment:
I would put the second granularity at least. It's definitely possible that one person or multiple people might run this test twice in a minute.